### DISCUSSION TITLE
Music videos question

### DISCUSSION THREAD

**Discussion by JabbaDaHutt762** [RandomNinjaAtk/docker-lidarr-extended#129] [Q&A] [answered]
I had a ton of garbage music videos in my collection and rather than try to sort through each i decided to delete the whole directory and allow lidarr-extended to start over with just the artists in my lidarr music directory.  reading the log though it seems to skip a ton because its "already been downloaded".  is there a way to reset that flag to go out and grab them again?

**Comment by RandomNinjaAtk** [comment, marked as answer]
These scripts detect previous downloads by either checking for the existing file, or log information.

So removing the relevant log files from `/config/extended/logs` (`/config/extended/logs/video/complete`) should do the trick....

Edit:

Also, pro tip, set `videoFormat` parameter to `bv[width>=1280]+ba` to limit your videos to 720p minimum resolution.

**Reply by JabbaDaHutt762** [reply]
> Also, pro tip, set videoFormat parameter to bv[width>=1280]+ba to limit your videos to 720p minimum resolution.

Im looking at the docker setup and i dont see that videoformat paremeter.   is it there or do i need to add that and if so how?

**Reply by RandomNinjaAtk** [reply]
Just add it as a parameter in your container config, same as the others.

**Comment by JabbaDaHutt762** [comment]
![variable](https://user-images.githubusercontent.com/57692223/202277428-26be0912-1e31-4ba3-b27a-3c4e985a3e1c.JPG)

Is this right?

**Reply by RandomNinjaAtk** [reply]
Looks like you didn’t add the key for it. Edit one of the other parameters to see what it should look like.

The value and name look fine.

**Reply by JabbaDaHutt762** [reply]
![video format](https://user-images.githubusercontent.com/57692223/202279997-5697d71e-9b24-4103-984e-f64f5c5bf202.JPG)

**Reply by JabbaDaHutt762** [reply]
awesome.  one more question.  how do you log into youtube to avoid this error?
![youtube](https://user-images.githubusercontent.com/57692223/202280666-bc5acdfc-f621-46fa-88fa-efb0d47df584.JPG)

**Reply by RandomNinjaAtk** [reply]
https://github.com/RandomNinjaAtk/docker-lidarr-extended/discussions/127#discussioncomment-4145376

**Reply by JabbaDaHutt762** [reply]
ahh ok.  i created the cookie file and dropped it in the config folder but the rest of the language is beyond my understanding.   i appreciate your help and time bud.

**Reply by RandomNinjaAtk** [reply]
No problem! Hopefully it runs smoothly after the few tweaks.

**Reply by JabbaDaHutt762** [reply]
is there a way to direct the video downloads through a download client?  so they are visible without watching the log file?  and resumeable?

**Reply by RandomNinjaAtk** [reply]
No, there is no web interface for yt-dlp or any of the scripts used in this container. You can view the logs via the web, they are located in the lidarr web interface under the log file section.
